supply-chain/platform-engineering/how-to/starter-supply-chains.hbs.md

```diff
@@ -0,0 +1,91 @@
+  --component conventions-1.0.0 \
+  --component app-config-web-1.0.0 \
+  --component carvel-package-1.0.0 \
+  --component deployer-1.0.0
+```
+
+> **Note** In order to deploy other workload types, replace the **app-config-web-1.0.0** component with other options such as **app-config-server-1.0.0** or **app-config-worker-1.0.0**
+
+## Build an application and store the artifact in Git
+
+This Supply Chain will build a Carvel package from application source and store it in a git repository for deployment to a runtime environment.
+
+It will perform the following actions:
+
+* Pull application source from Git
+* Build a container image using Buildpacks
+* Generate a runtime definition with the image
+* Generate a Knative service
+* Generate a Carvel package to make the application deployable
+* Create a PR against a git repository with the Carvel package contents
+
+Run the following command to generate the supply chain. Ensure you have first initialized a working directory using [tanzu supplychain init](../../reference/supplychain-cli/tanzu_supplychain_init.hbs.md).
+
+```
+tanzu supplychain generate \
+  --kind CarvelPackage \
+  --description "Build an application from source and store the Carvel package in Git" \
+  --component source-git-provider-1.0.0 \
+  --component buildpack-build-1.0.0 \
+  --component conventions-1.0.0 \
+  --component app-config-web-1.0.0 \
+  --component carvel-package-1.0.0 \
+  --component git-writer-pr-1.0.0 
+```
+
+> **Note** In order to write directly to a git repository without creating a PR, replace the **git-writer-pr-1.0.0** component with **git-writer-1.0.0**. See [Deploy an application package from Git](#deploy-an-application-package-from-git) next for a Supply Chain that will deploy the package that is generated by this Supply Chain.
+
+## Deploy an application package from Git
+
+This Supply Chain will deploy a Carvel package from a git repository.
+
+It will perform the following actions:
+
+* Pull application package from Git
+* Translate the Carvel package to a deployable package
+* Deploy the application
+
+Run the following command to generate the supply chain. Ensure you have first initialized a working directory using [tanzu supplychain init](../../reference/supplychain-cli/tanzu_supplychain_init.hbs.md).
+
+```
+tanzu supplychain generate \
+  --kind PackageDeploy \
+  --description "Deploy a Carvel package from Git" \
+  --component source-git-provider-1.0.0 \
+  --component source-package-translator-1.0.0 \
+  --component deployer-1.0.0
+```
+
+## Coming from Supply Chain Choreographer
+
+For users coming from Supply Chain Choreographer, the recipes in this document are analogous to OOTB Supply Chains and Profile experiences. Use this mapping to help decide which recipe you should start with. These recipes do not provide exact parity with OOTB Supply Chains, nor will they. Recipes will help users get started so they can then author Supply Chains that are appropriate for their needs.
+
```

Comment:
Supply Chain Basic and Delivery Basic are SCC supply chains, but Iterate Profile is an installation profile.
Is this correct - as the lead in sentence makes me think this is a mapping between SCC and TSC?

Reply:
yes that's correct.  this is for users coming from carto supply chains and moving into this new supply chain beta. the maping will make sense to folks with that previous carto supply chain context